File: Programming Basics with JS/T04 - Nested Conditional Statements/exercise/07. Hotel Room.js
function hotelRoom(input) {
    let month = input[0];
    let stay = Number(input[1]);
    let priceForStudio = 0;
    let priceForApart = 0;

    if (stay > 7 && stay <= 14){

        switch(month){
            case "May" :
            case "October" :
                priceForStudio = 50 * 0.95 * stay;
                priceForApart = 65 * stay;
                break;
            case "June" :
            case "September" :
                priceForStudio = 75.20 * stay;
                priceForApart = 68.70 * stay;
                break;
            case "July" :
            case "August" :
                priceForStudio = 76 * stay;
                priceForApart = 77 * stay;
                break;
        }

    } else if (stay > 14) {

        switch(month) {
            case "May" :
            case "October" :
                priceForStudio = 50 * 0.70 * stay;
                priceForApart = 65 * 0.90 * stay;
                break;
            case "June" :
            case "September" :
                priceForStudio = 75.20 * 0.80 * stay;
                priceForApart = 68.70 * 0.90 * stay;
                break;
            case "July" :
            case "August" :
                priceForStudio = 76 * stay;
                priceForApart = 77 * 0.90 * stay;
                break;
        }

    } else {

        switch(month) {
            case "May" :
            case "October" :
                priceForStudio = 50 * stay;
                priceForApart = 65 * stay;
                break;
            case "June" :
            case "September" :
                priceForStudio = 75.20 * stay;
                priceForApart = 68.70 * stay;
                break;
            case "July" :
            case "August" :
                priceForStudio = 76 * stay;
                priceForApart = 77 * stay;
                break;
        }
    }
    console.log(`Apartment: ${priceForApart.toFixed(2)} lv.`);
    console.log(`Studio: ${priceForStudio.toFixed(2)} lv.`);
}
hotelRoom(["June", "14"]);

File: Programming Basics with Go/T04 - Conditional Statements (Advanced)/exercise/07. Hotel Room.go
package main

import "fmt"

func main() {
	var month string
	fmt.Scanln(&month)

	var count int
	fmt.Scanln(&count)

	var totalStudio float32 = 0
	var totalApartment float32 = 0

	switch month {
		case "May", "October":
			totalStudio = float32(count) * 50
			totalApartment = float32(count) * 65

			if count > 7 && count < 15 {
				totalStudio = totalStudio * 0.95
			} else if count > 14 {
				totalStudio = totalStudio * 0.7
				totalApartment = totalApartment * 0.9
			}
			break
		case "June", "September":
			totalStudio = float32(count) * 75.2
			totalApartment = float32(count) * 68.7
			if count > 14 {
				totalStudio = totalStudio * 0.8
				totalApartment = totalApartment * 0.9
			}
			break
		case "July", "August":
			totalStudio = float32(count) * 76
			totalApartment = float32(count) * 77
			if count > 14 {
				totalApartment = totalApartment * 0.9
			}
	}
	fmt.Printf("Apartment: %.2f lv.\n", totalApartment)
	fmt.Printf("Studio: %.2f lv.", totalStudio)
}
